server/Arcadia.Assistant.Web/Controllers/DownloadWebController.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Arcadia.Assistant.Web.Controllers
+{
+    using System.IO;
+    using System.Linq;
+
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.AspNetCore.Mvc;
+
+    using Arcadia.Assistant.Web.Configuration;
+    using Arcadia.Assistant.Web.Models;
+
+    [Route("/download")]
+    public class DownloadWebController : Controller
+    {
+        private readonly IDownloadApplicationSettings downloadApplicationSettings;
+        private readonly IHostingEnvironment hostingEnvironment;
+
+        public DownloadWebController(
+            IDownloadApplicationSettings downloadApplicationSettings,
+            IHostingEnvironment hostingEnvironment)
+        {
+            this.downloadApplicationSettings = downloadApplicationSettings;
+            this.hostingEnvironment = hostingEnvironment;
+        }
+
+        [HttpGet]
+        public IActionResult Index()
```

Comment:
better not to return an actual path but have an action that returns File action result.

Reply:
also, what do you think about having a file path stored in actor once it downloaded it? So we don't have paths in two places?

Reply:
and you won't need file pattern in config

Reply:
Done.